modules/custom/social_lazy_loading/js/lazy_load_reactivate.js

```diff
@@ -0,0 +1,20 @@
+(function ($) {
+
+  'use strict';
+
+  Drupal.behaviors.socialLazyLoadReactivate = {
+    attach: function (context, setting) {
+      var $accordion = $('.paragraph--type--accordion', context);
+
+      $accordion.each(function () {
+        var $currentAccord = $(this);
+        var $accordItem = $currentAccord.find('.paragraph--type--accordion-item');
+        $accordItem.find('.card__title-accord').on('click', function () {
+            var bLazy = new Blazy();
+            bLazy.revalidate();
+        });
+      });
+    }
```

Comment:
I think we can just collapse the searching done here into a single query:
```
$('.paragraph--type--accordion .paragraph--type--accordion-item .card__title-accord', context).on('click', function () {
  var bLazy = new Blazy();
  bLazy.revalidate();
});
```

:)